tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -0,0 +1,1263 @@
+        })
+      })
+
+      await page.goto('/search/granules?p=C194001210-LPDAAC_ECS&ot=2015-01-03T00:00:00.000Z,2015-01-03T23:59:59.999Z&tl=1420268129.401!2!1420243200!1420329599.999')
+
+      // Ensure the correct number of results were loaded
+      await testResultsSize(page, cmrHits)
+
+      // Timeline is correct
+      // Zoom level
+      await expect(page.locator('.edsc-timeline-tools > :nth-child(1)')).toHaveText('Day')
+      // Focused Date
+      await expect(page.locator('.edsc-timeline-tools__section--horizontal')).toHaveText('03 Jan 2015')
+      // Position
+      await expect(page.locator('.edsc-timeline-interval__interval-section-label').filter({ hasText: 'Jan 2015' })).toBeVisible()
+    })
+  })
+
+  test.describe('When the path is loaded with a focused granule', () => {
+    test('loads with the granule focused', async ({ page }) => {
+      const conceptId = 'C194001210-LPDAAC_ECS'
+      const cmrHits = 275361
+
+      await page.route('**/search/granules.json', (route) => {
+        const request = route.request()
+        const body = request.postData()
+        expect(body).toBe('echo_collection_id=C194001210-LPDAAC_ECS&page_num=1&page_size=20&sort_key=-start_date')
+        route.fulfill({
+          body: JSON.stringify(focusedGranuleGranulesBody),
+          headers: {
+            ...commonGranulesHeaders,
+            'access-control-expose-headers': 'cmr-hits',
+            'cmr-hits': cmrHits
+          }
+        })
+      })
+
+      await page.route('**/search/granules/timeline', (route) => {
+        const request = route.request()
+        const body = request.postData()
+        if (body) {
+          expect(body).toBe('end_date=2027-01-01T00:00:00.000Z&interval=day&start_date=2022-01-01T00:00:00.000Z&concept_id[]=C194001210-LPDAAC_ECS')
+        }
+
+        route.fulfill({
+          body: JSON.stringify(focusedGranuleTimelineBody),
+          headers: focusedGranuleTimelineHeaders
+        })
+      })
+
+      await page.route('**/api', (route) => {
+        const body = JSON.stringify(route.request().postData())
+
+        if (body === JSON.stringify(graphQlGetCollection(conceptId))) {
+          route.fulfill({
+            body: JSON.stringify(focusedGranuleCollectionGraphQlBody),
+            headers: focusedGranuleGraphQlHeaders
+          })
+        } else if (body === '{"query":"\\n    query GetGranule(\\n      $params: GranuleInput\\n    ) {\\n      granule(\\n        params: $params\\n      ) {\\n        granuleUr\\n        granuleSize\\n        title\\n        onlineAccessFlag\\n        dayNightFlag\\n        timeStart\\n        timeEnd\\n        dataCenter\\n        originalFormat\\n        conceptId\\n        collectionConceptId\\n        spatialExtent\\n        temporalExtent\\n        relatedUrls\\n        dataGranule\\n        measuredParameters\\n        providerDates\\n      }\\n    }","variables":{"params":{"conceptId":"G2058417402-LPDAAC_ECS"}}}') {
```

Comment:
This string feels like it should be a separate file but, I know it isn't on the existing cypress test and I we already have alot of mocks so I'm fine either way